apps/web/components/pages/task/ChildIssueCard.tsx

```diff
@@ -0,0 +1,157 @@
+
+export const ChildIssueCard = () => {
+	const { trans } = useTranslation();
+	const modal = useModal();
+
+	const task = useRecoilValue(detailedTaskState);
+	const { tasks } = useTeamTasks();
+	const [hidden, setHidden] = useState(false);
+
+	const childTasks = useMemo(() => {
+		console.log(task?.children);
+
+		return (
+			task?.children
+				?.map((t) => {
+					return tasks.find((ts) => ts.id === t.id) || t;
+				})
+				.filter(Boolean) || []
+		);
+	}, [task, tasks]);
+
+	return (
+		<Card
+			className="w-full pt-0 px-4 md:pt-0 md:px-4 dark:bg-[#25272D] flex flex-col gap-[1.125rem] border border-[#00000014] dark:border-[#26272C]"
+			shadow="bigger"
+		>
+			<div className="flex justify-between items-center gap-5 py-2 border-b border-b-[#00000014] dark:border-b-[#7B8089]">
+				<p className="text-base font-semibold">{trans.common.CHILD_ISSUES}</p>
+
+				<div className="flex items-center justify-end gap-2.5">
+					<div className="border-r border-r-[#0000001A] flex items-center gap-2.5">
+						<span onClick={modal.openModal}>
+							<PlusIcon className="h-7 w-7 stroke-[#B1AEBC] dark:stroke-white cursor-pointer" />
+						</span>
+					</div>
+
+					<button onClick={() => setHidden((e) => !e)}>
+						{hidden ? (
+							<ChevronDownIcon className="h-4 w-4 stroke-[#293241] dark:stroke-white cursor-pointer" />
+						) : (
+							<ChevronUpIcon className="h-4 w-4 stroke-[#293241] dark:stroke-white cursor-pointer" />
+						)}
+					</button>
+				</div>
+			</div>
+
+			{childTasks.length > 0 && (
+				<div
+					className={clsxm(
+						'flex flex-col max-h-80 gap-3',
+						hidden && ['hidden']
+					)}
+				>
+					{childTasks.map((task) => {
+						return (
+							<TaskLinkedIssue
+								key={task.id}
+								task={task}
+								className="dark:bg-[#25272D] py-0"
+							/>
+						);
+					})}
+				</div>
+			)}
+
+			{task && <CreateChildTask task={task} modal={modal} />}
+		</Card>
+	);
+};
+
+function CreateChildTask({
+	modal,
+	task,
+}: {
+	modal: IHookModal;
+	task: ITeamTask;
+}) {
+	const { trans } = useTranslation();
+
+	const { tasks, loadTeamTasksData } = useTeamTasks();
+	const { updateTask } = useTeamTasks();
+
+	const [loading, setLoading] = useState(false);
+
+	const onTaskSelect = useCallback(
+		async (childTask: ITeamTask | undefined) => {
+			if (!childTask) return;
+
+			setLoading(true);
+
+			await updateTask({
+				...childTask,
+				parentId: task.id,
+				parent: task,
+			});
+
+			loadTeamTasksData(false).finally(() => {
+				setLoading(false);
+				modal.closeModal();
+			});
+		},
+		[task, updateTask, loadTeamTasksData, modal]
+	);
+
+	const childTasks = task.children?.map((t) => t.id) || [];
+
+	const unchildTasks = tasks.filter((t) => {
+		return (
+			t.id !== task.id &&
+			!childTasks.includes(t.id) &&
+			!['Epic', 'Story'].includes(t.issueType)
+		);
+	});
+
+	return (
+		<Modal isOpen={modal.isOpen} closeModal={modal.closeModal}>
+			<div className="w-[98%] md:w-[668px] relative">
```

Comment:
@paradoxe35 can we use `rem` instead of `px` here? Lets shift to `rem` in whole projects gradually for old code. and new code we use rem only.